src/FoundryCSS/ui/blocks/Card/_card.scss

```diff
@@ -5,24 +5,40 @@
  * horizontally or vertically.
  */
 
-@import 'card-config';
+@import "card-config";
 
 %card,
 .card {
+    position: relative;
     background-color: $card-background-color;
     border-radius: $card-border-radius;
-    padding: map-get($card-sizes, 'default');
+    padding: map-get($card-sizes, "default");
     box-shadow: $card-shadow;
 }
 
+%card__hitbox,
+.card__hitbox {
+    z-index: 2;
+
+    &:before {
+        content: "";
+        position: absolute;
+        top: 0;
+        left: 0;
+        width: 100%;
+        height: 100%;
+        z-index: 2;
+    }
+}
+
```

Comment:
Still not a huge fan of this hitbox pattern. It prevents you selecting the content of the card. What advantage does it have over wrapping the card or it's contents in an `<a>` tag?

Reply:
I don't think there's an issue with <a> tags, but I don't think <button> elements allow certain types of other element otherwise the markup will be invalid? According to Mozilla it suggests that only 'phrasing' content is allowed within - so that even means <div> is out of the picture :s